core/src/uix/compiler/alpha.cljs

```diff
@@ -41,25 +41,12 @@
 (defn add-transform-fn [f]
   (swap! transform-fns conj f))
 
-(defn ^string capitalize [^string s]
-  (if (< (.-length s) 2)
-    (str/upper-case s)
-    (str ^string (str/upper-case (subs s 0 1)) ^string (subs s 1))))
-
 (defn ^string dash-to-camel [dashed]
-  (let [name-str (-name dashed)
-        parts (.split name-str #"-")
-        ^string start (aget parts 0)
-        parts (.slice parts 1)]
-    (if (or (= start "aria") (= start "data"))
-      name-str
-      (str start (-> parts
-                     (array-reduce
-                       (fn [a p]
-                         (.push a (capitalize p))
-                         a)
-                       #js [])
-                     ^string (.join ""))))))
+  (let [name-str (-name dashed)]
+    (cond
+      (some? (re-matches #"^(aria-|data-).*" name-str)) name-str
```

Comment:
Attributes transformation is one of the hot paths in the library, I'm usually trying to make this code as fast as possible. Which means using native JS methods to manipulate strings in this case. I can rewrite these later if you don't feel like doing it.

Reply:
Please rewrite it later however you feel. I agree that it might be a bit faster if google closure compiler does not fully inline core functions in this case.